recipes-containers/kubelet/kubelet_git.bb

```diff
@@ -22,8 +22,8 @@ SYSTEMD_AUTO_ENABLE_${PN} = "enable"
 SRCREV = "83b266ae6939012883611d6dbda745f2490a67c4"
 PR = "r1"
 
-DEPENDS = "libseccomp bash-native"
-RDEPENDS_${PN} += " docker libseccomp cni bash jq"
```

Comment:
@x85446 FYI, `jq` is used by launch-kubelet.sh - https://github.com/PelionIoT/meta-pelion-edge/blob/dev/recipes-containers/kubelet/files/launch-kubelet.sh.